### PR TITLE
Tokenize hexadecimal string sequences + minor enhancements

### PR DESCRIPTION
**Pull request by MaxGraey**
Allow "\xFD\x7F" sequences for string literals

### CODE REVIEW

**Comment by MaxGraey** [comment]
JavaScript also support octal sequences but its not recommended so not sure we should implement it so only hexadecimal for now

File: src/tokenizer.ts [hunk 1]
```diff
@@ -1537,8 +1534,7 @@ export class Tokenizer extends DiagnosticEmitter {
     throw new Error("not implemented"); // TBD
   }
 
-  readUnicodeEscape(): string {
-    var remain = 4;
+  private readSequenceEscape(remain: i32): string {
```
Comment (dcodeIO) [inline src/tokenizer.ts]: Perhaps `readEscapeSequence`?

Comment (MaxGraey) [reply src/tokenizer.ts]: Thanks, that's better

**Comment by dcodeIO** [comment]
LGTM :)

File: src/tokenizer.ts (hunk 1 above)
Comment (MaxGraey) [reply src/tokenizer.ts]: Unfortunately this name already used

Comment (dcodeIO) [reply src/tokenizer.ts]: Then perhaps just `readHexadecimalEscape`, which both old `readHexadecimalEscape` and `readUnicodeEscape` are, and remove the old ones - specifying `remain`.

Comment (MaxGraey) [reply src/tokenizer.ts]: Done!

**Comment by dcodeIO** [comment]
Great, thanks!

**Pull request merged**